tests/test_models.py

```diff
@@ -538,6 +538,10 @@ def _test_score(estimator, p_mat, graph):
     np.random.seed(8888)
     graph = graph.copy()
     p_mat = p_mat.copy()
+    
+   if np.shape(p_mat)!=np.shape(graph):
```

Comment:
This is not the test we are looking for. Tests make sure that our code works when it is supposed to work, and throws errors when it is supposed to throw errors. In this case, we want it to throw an error when `p_mat` and `graph` are different sizes. See [here](https://github.com/neurodata/graspy/blob/324764b267449585a5105a97c66dcf77c169193d/tests/test_models.py#L28) for examples.

Reply:
Have updated, please check it again.